package.json

```diff
@@ -34,6 +34,7 @@
     "elliptic": "^6.5.4",
     "ethereumjs-util": "^7.1.2",
     "express": "^4.17.1",
+    "express-async-errors": "^3.1.1",
```

Comment:
this is to automatically catch async errors.
No need to use `wrapHandler` anymore